internal/uploader/dummy.go

```diff
@@ -0,0 +1,26 @@
+package uploader
+
+import (
+	"context"
+
+	"github.com/go-openapi/strfmt"
+	eventsapi "github.com/openshift/assisted-service/internal/events/api"
+	"github.com/sirupsen/logrus"
+)
+
+type DummyUploader struct {
+	log logrus.FieldLogger
+}
+
+var _ UploadHandler = &DummyUploader{}
+
+func NewDummyUploader(log logrus.FieldLogger) *DummyUploader {
```

Comment:
Probably don't need this either if the interface "constructor" is returning this.

Reply:
Good point, removed this function!